code/modules/research/xenoarchaeology/artifact/effect_old.dm

```diff
@@ -297,7 +297,7 @@
 						continue
 					to_chat(M, "<span class='warning'>A wave of energy overwhelms your senses!</span>")
 					M.paralysis += 3
-					M.weakened += 4
+					M.knockdown += 4
```

Comment:
You get the idea by now. There's 18 cases of knockdown += in here, not including the one that's commented out anyway